app/views/posts/index.html.erb

```diff
@@ -45,5 +45,9 @@
     <button class="back-all-users">
       <%= link_to "Back to all users", root_path %>
     </button>
```

Comment:
- [ ] Great effort working on the project milestone but currently I could not access the form for posts and comments which makes it difficult to create either post or comment. I would kindly suggest you work on the error message as i have recorded a video to demonstrate how the app is currently behaving.

Video demonstration 👇 

[screen-capture (3).webm](https://github.com/ragangithub/Blog_app/assets/78634841/b08925ad-8aa0-435d-9db8-4aa374a51c65)